packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.types.ts

```diff
@@ -57,9 +57,9 @@ export interface IFocusZoneProps extends React.HTMLAttributes<HTMLElement | Focu
   disabled?: boolean;
 
   /**
-   * Element type the root element will use. Default is "div".
+   * A component that should be used as the root element of the link returned from the Link component.
    */
-  elementType?: keyof React.ReactHTML;
```

Comment:
We'll need to deprecate `elementType` first and not remove it until next major release else we risk breaking backwards-compatibility.